inference-engine/src/plaidml_plugin/ops/variadic_split.cpp

```diff
@@ -11,6 +11,22 @@
 #include "plaidml/op/op.h"
 
 using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace {
+
+template <typename T>
+std::vector<T> cast_constant_operand(size_t operand_idx, ngraph::Node* layer) {
+  auto* ngraph_const = ngraph::as_type<ngraph::op::Constant>(layer->get_input_node_ptr(operand_idx));
+  if (ngraph_const) {
+    return ngraph_const->cast_vector<T>();
+  } else {
+    THROW_IE_EXCEPTION << "Dynamic padding not currently supported by PlaidML plugin; all of pads_begin, pads_end, "
+                          "and pads_value must be Constants";
```

Comment:
The error here should be about how dynamic `split_lengths` cannot be used with variadic split; the parameters mentioned here aren't used by variadic split.

Or something generic about dynamic output shape not being supported would be fine too.